core/alert/handler/handler.ex

```diff
@@ -36,7 +39,9 @@ defmodule AntikytheraCore.Alert.Handler do
   - Messages received by handlers will be sent out in "fast-then-delayed" pattern:
       - Occasional messages will be flushed from the buffer and sent out in `fast_interval`.
       - Messages arriving too frequently will be buffered for `delayed_interval` until they are sent out.
-  - By default, `fast_interval` is #{@default_fast_interval} seconds and `delayed_interval` is #{@default_delayed_interval} seconds.
+  - By default, `fast_interval` is #{@default_fast_interval} seconds and `delayed_interval` is #{
+    @default_delayed_interval
```

Comment:
Is the reason of this multi line that this line is too long?

Reply:
>I guess that the reason why this becomes multi lines is the original line is too long.

I think so.
According to [here](https://github.com/elixir-lang/elixir/blob/4fea9cc942f632d5a1692eb85774d097546bc4e5/lib/elixir/lib/code.ex#L426) , `{}` seems to be the only place in which `formatter` can introduce a new line safely.

Reply:
I prefer enforcing developers to write document lines shorter than the threshold length rather than accepting such a line break. (at least, lines including interpolation)